src/pipeline.cc

```diff
@@ -846,13 +846,49 @@ class PipelineWorker : public Nan::AsyncWorker {
           if (isDzZip) {
             baton->tileContainer = VIPS_FOREIGN_DZ_CONTAINER_ZIP;
           }
+          // Forward format options through suffix
+          std::string suffix;
+          if (baton->tileFormat == "png") {
+            suffix = ".png[interlace="
```

Comment:
Might it be worth externalising this string concatenation? I'm thinking of something like a function that accepts a `vector<pair<string, string>>` and returns a single `string`.